jdbc-adapter/virtualschema-jdbc-adapter/src/main/java/com/exasol/adapter/jdbc/JdbcMetadataReader.java

```diff
@@ -242,8 +242,8 @@ private static String findSchema(final String schema, final DatabaseMetaData dbM
     }
 
     private static List<TableMetadata> findTables(final String catalog, final String schema,
-            final List<String> tableFilter, final DatabaseMetaData dbMeta, final SqlDialect dialect,
-            final JdbcAdapterProperties.ExceptionHandlingMode exceptionMode) throws SQLException {
+                                                  final List<String> tableFilter, final DatabaseMetaData dbMeta, final SqlDialect dialect,
+                                                  final JdbcAdapterProperties.ExceptionHandlingMode exceptionMode, List<String> ignoreErrorList) throws SQLException {
```

Comment:
Method to long and complex. Refactor to break into smaller blocks (e.g. with "extract method").

Reply:
Maybe use Parameter Object in a refactoring branch